src/man/sssd.conf.5.xml

```diff
@@ -211,6 +216,13 @@
                                 <phrase condition="with_pac_responder">, pac</phrase>
                                 <phrase condition="with_ifp">, ifp</phrase>
                             </para>
+                            <para>
+                                <phrase condition="have_systemd">
+                                    By default, all services are disabled and the administrator
+                                    must enable the ones allowed to be used by executing:
+                                    "systemctl enable sssd-@service@.socket".
```

Comment:
I do think so. Shall I dig into their code in order to provide a patch or just opening a bug for them would be enough?